fla/layers/mom.py

```
@@ -0,0 +1,613 @@
+        o = self.o_proj(o)
+
+        return o, None, past_key_values, router_logits
+
+
+    def shared_o(
+        self,
+        hidden_states: torch.Tensor,
+        attention_mask: Optional[torch.Tensor] = None,
+        recurrent_state = None,
+        use_cache: Optional[bool] = False,
+        conv_state_q = [None, None],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)